### DISCUSSION TITLE
Welcome to thacer Discussions!

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by francoiscrouzet** [comment]
@AlZorba voila, les discussions sont activées

**Reply by francoiscrouzet** [reply]
discussion en français, issues et tout le reste en anglais?

**Reply by AlZorba** [reply]
perque no ?!
en fait la numérotation # est incrémentielle quelque soit le truc? discus, issues, PR etc ?

**Comment by francoiscrouzet** [comment]
Ah en fait je crois que les discussions c'est pour papoter, poser des questions techniques, et discuter de choses dont on est pas sûr qu'on va les faire. Et seulement si oui, alors il y a la fonction `create issue from discussion`

**Reply by francoiscrouzet** [reply]
ah, et on peut aussi `lock the conversation`, quand on a lu et rien a dire de plus ?

**Comment by AlZorba** [comment]
This is over !
